tests/pytest/run_files/job_jenkins_sarus

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -l
+#SBATCH --job-name="FV3gfs_800km_resolution"
+#SBATCH --partition=cscsci
+#SBATCH --constraint=gpu
+#SBATCH --time=00:04:00
+#SBATCH --nodes=1
+#SBATCH --ntasks=6
+#SBATCH --hint=nomultithread
+
+##
+## Set the compute node environment
+##
+
+module load daint-gpu sarus
```

Comment:
This would change as well if you were to run on the multi-core partition of Daint: module load daint-mc sarus